rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -490,8 +504,7 @@ maas_rpc_scripts_dir: /opt/rpc-openstack/scripts
 #
 maas_excluded_checks: []
 
-
-#
+-#
```

Comment:
This looks to be causing the checks to fail. Is it simply a typo or are you trying to do something with it?